### PR TITLE
Added some synonymous terms for `covers` and `skip_words`

### PR DESCRIPTION
**Pull request by LorenzKahl**
Spelling References:
- https://www.duden.de/rechtschreibung/Markise
- https://de.wikipedia.org/wiki/Raffstore

### CODE REVIEW

File: sentences/de/_common.yaml [hunk 1]
```diff
@@ -54,8 +54,8 @@ expansion_rules:
   schliessen: "(schließ | schließe)"
   machen: "(mach | mache)"
   setzen: "(setze | stelle | ändere)"
-  abdeckung: "(das Rollo | die Rollos | die Abdeckung | die Abdeckungen | den Rollladen | die Rollläden | die (Jalousie | Jalousien))"
-  alle_abdeckungen: "alle (Rollos | Abdeckungen | Rollläden | Jalousien)"
+  abdeckung: "(das Rollo | die Rollos | die Abdeckung | die Abdeckungen | den Rollladen | die Rollläden | die (Jalousie | Jalousien) | den Raffstore | die Raffstores | die Markise | die Markisen)"
```
[the remark this answers is not in the document]
Comment (LorenzKahl) [reply sentences/de/_common.yaml:57]: @mib1185 thanks. Then we should also apply that to ’die (Abdeckung | Abdeckungen)’, too, right?

[the remark this answers is not in the document]
Comment (LorenzKahl) [reply sentences/de/_common.yaml]: Updated both! Thanks!

**Comment by mib1185** [comment]
as long as @Scorpoon is the only language lead for German, he should still approve this pr, too 🙂

**Review by Scorpoon — APPROVED** [review]
Looks good, thanks for your contribution 😊

**Pull request merged**